tracking/src/SessionManager.js

```diff
@@ -0,0 +1,91 @@
+
+const SESSION_TIMEOUT = 1800000; 
+const DAY_IN_MS = 86400000; 
+const KEY_UID = "va-uid"; 
+const KEY_SID = "va-key-sid";
+const KEY_SID_TIMESTAMP = "va-key-sid-timestamp"; 
+
+class SessionManager {
+    constructor(uid) {
+
+        this.uid = (uid === undefined) ? this.getUID() : uid; 
```

Comment:
should be uid ?. Blank is also not accepted.

Reply:
changed